acos_client/v30/slb/server.py

```diff
@@ -23,11 +23,12 @@ class Server(base.BaseV30):
     def get(self, name, **kwargs):
         return self._get(self.url_prefix + name, **kwargs)
 
-    def create(self, name, ip_address, status=1, **kwargs):
+    def create(self, name, ip_address, conn_limit, status=1, **kwargs):
```

Comment:
conn_limit should have a default, and be optional (it's default should be the current value), and it should come *after* the last named argument, so you don't break argument ordering backwards compatability.